thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdeyeSmtpConfiguration.java

```diff
@@ -0,0 +1,89 @@
+package com.linkedin.thirdeye.anomaly;
+
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+public class ThirdeyeSmtpConfiguration {
+  private String fromAddress;
+  private String toAddresses;
```

Comment:
clarify that from/to are for the application notifications (eg function failures).